docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/set-metadata.md

```diff
@@ -0,0 +1,121 @@
+---
+sidebar_label: set-metadata
+description: Updates metadata on Stage or Freight resources during the promotion process.
+---
+
+# `set-metadata`
+
+`set-metadata` updates metadata on `Stage` or `Freight` resources during the
+promotion process. This step allows you to attach arbitrary key-value pairs to the
+status of these resources, which can be useful for tracking deployment information,
```

Comment:
Reword this somehow, please to not use the word "deployment." Kargo does not deploy.

```suggestion
status of these resources, which can be useful for tracking deployment information,
```